lib/prima_auth0_ex/application.ex

```diff
@@ -41,14 +55,25 @@ defmodule PrimaAuth0Ex.Application do
     end
   end
 
-  defp cache_enabled?, do: Application.get_env(:prima_auth0_ex, :client, cache_enabled: false)[:cache_enabled]
-  defp client_configured?, do: Application.get_env(:prima_auth0_ex, :client) != nil
+  defp cache_enabled?,
+    do: Application.get_env(:prima_auth0_ex, :redis, enabled: false)[:enabled]
```

Comment:
I asked myself the same thing... probably...? I'll see what that would entail

Reply:
For now I think we can keep it as a global option and implement this in the future if it's needed

Reply:
Yeah I agree on keeping it global because it's simpler and maybe in the future we won't need multiple clients anymore so who knows :man_shrugging: 

Also I don't really see having mixed cached and non cached clients, even though it may be possible